src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -531,7 +531,8 @@ public override void _Process(float delta)
         {
             if (!CurrentGame.GameWorld.IsAutoEvoFinished())
             {
-                LoadingScreen.Instance.Show("Loading Microbe Editor", "Waiting for auto-evo: " +
+                LoadingScreen.Instance.Show(TranslationServer.Translate("MICROBE_EDITOR_LOADING"),
+                    TranslationServer.Translate("MICROBE_EDITOR_WAITING_AUTO_EVO") + " " +
```

Comment:
waiting for auto-evo should be generic named, because it will 100% be used for other stages as well, so then we could have silly translations where at some point the same exact auto evo running message stops being translated half way through the game.